core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -136,6 +136,7 @@ registry.ci.openshift.org/origin/4.10:ibm-cloud-controller-manager quay.io/opens
 registry.ci.openshift.org/origin/4.10:ibm-vpc-block-csi-driver quay.io/openshift/origin-ibm-vpc-block-csi-driver:4.10 quay.io/openshift/origin-ibm-vpc-block-csi-driver:4.10.0 quay.io/openshift/origin-ibm-vpc-block-csi-driver:latest
 registry.ci.openshift.org/origin/4.10:ibm-vpc-block-csi-driver-operator quay.io/openshift/origin-ibm-vpc-block-csi-driver-operator:4.10 quay.io/openshift/origin-ibm-vpc-block-csi-driver-operator:4.10.0 quay.io/openshift/origin-ibm-vpc-block-csi-driver-operator:latest
 registry.ci.openshift.org/origin/4.10:ibmcloud-machine-controllers quay.io/openshift/origin-ibmcloud-machine-controllers:4.10 quay.io/openshift/origin-ibmcloud-machine-controllers:4.10.0 quay.io/openshift/origin-ibmcloud-machine-controllers:latest
+registry.ci.openshift.org/origin/4.10:image-customization-controller quay.io/openshift/origin-image-customization-controller:4.10 quay.io/openshift/origin-image-customization-controller:4.10.0 quay.io/openshift/origin-image-customization-controller:latest
```

Comment:
What kind of image is this customizing?

Reply:
A CoreOS ISO or initramfs.

Reply:
I would probably have preferred this to be `machine-image-customization-controller` since it's not customizing generic images, correct?

Reply:
Today it's taking the off-the-shelf CoreOS image and adding an Ignition file customised with network config supplied via a k8s CR. So it's fairly generic, but not completely so in the sense that the image being customised has to be some flavour of CoreOS (RHCOS/FCOS).

You definitely have a good point that "machine-image" would make it more clear that it's customising a disk image rather than, say, a container image. There is a possibility that it could cause confusion with the controller for [CoreOS Layering](https://github.com/coreos/enhancements/blob/main/os/coreos-layering.md#coreos-layering) in the MCO (this is not expected to require its own container image, but avoiding naming conflicts with it was definitely something that came up in the staff engineer review for adding this image to the CVO payload).

To be honest though, my main worry is that we are a long way down the path already and there is a high risk of missing the release if we have to rename it.

Reply:
I'd probably say from a name approval and review process it sounds like this should be `machine-image-customization-controller`, and if you ship with it this release you'll need to rename next release.  It's usually more expensive to rename later, but not that much more.

Reply:
Happy to rename in the next release.

Reply:
Ok, link whatever work item tracks that here and @ me

Reply:
@smarterclayton I created https://issues.redhat.com/browse/METAL-116 and assigned it to myself.